### PR TITLE
Updating a RWA Jar Address

### PR DESCRIPTION
**Pull request by K-Lascar**
Hi Dune Team and Contributors, I am apart of the Steakhouse team, we identified an address within the maker accounting ethereum, and have replaced it with the correct address.

### CODE REVIEW

File: models/maker/ethereum/maker_ethereum_accounting.sql
Comment (Hosuke) [inline models/maker/ethereum/maker_ethereum_accounting.sql:1]: Hi @K-Lascar,
we may need to migrate this spell into dune sql format.

**Review by Hosuke — CHANGES_REQUESTED** [review]

File: models/maker/ethereum/maker_ethereum_accounting.sql
Comment (K-Lascar) [reply models/maker/ethereum/maker_ethereum_accounting.sql:1]: Hey Hosuke, sure a thing will take a look at revising it into dune sql format.

Comment (K-Lascar) [reply models/maker/ethereum/maker_ethereum_accounting.sql:1]: Fixed Hosuke, let me know if we're good

**Comment by Hosuke** [comment]
> **CLA Assistant Lite bot:** Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/duneanalytics/spellbook/blob/main/CLA.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.
> 
> I have read the CLA Document and I hereby sign the CLA
> 
> You can retrigger this bot by commenting **recheck** in this Pull Request

Hi @K-Lascar,
you may need to follow instructions here and sign the license agreement before merging.

File: models/maker/ethereum/maker_ethereum_accounting.sql
Comment (Hosuke) [reply models/maker/ethereum/maker_ethereum_accounting.sql:1]: Do you want to migrate this spell? As legacy spells will be deprecated soon.

**Review by jeff-dude — APPROVED** [review]

**Comment by jeff-dude** [comment]
gm @K-Lascar thanks for helping contribute in spellbook!

as hosuke mentioned, plz post a gh comment on this PR with the one sentence from the bot above in order to sign the spellbook license agreement. we can't merge until done :)

**Comment by K-Lascar** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by K-Lascar** [comment]
Thanks Jeff and Hosuke for your help

**Comment by jeff-dude** [comment]
@Hosuke do you mind migrating this to dunesql in a separate PR?

that way we can keep the history here of a clean edit to legacy file, then we will have a clean history of migration as well

you could pull over to new branch, then start new PR, then revert changes here?

**Comment by Hosuke** [comment]
> @Hosuke do you mind migrating this to dunesql in a separate PR?
> 
> that way we can keep the history here of a clean edit to legacy file, then we will have a clean history of migration as well
> 
> you could pull over to new branch, then start new PR, then revert changes here?

I have recovered `models/maker/ethereum/maker_ethereum_accounting.sql` from the main branch.

**Comment by jeff-dude** [comment]
> > @Hosuke do you mind migrating this to dunesql in a separate PR?
> > that way we can keep the history here of a clean edit to legacy file, then we will have a clean history of migration as well
> > you could pull over to new branch, then start new PR, then revert changes here?
> 
> I have recovered `models/maker/ethereum/maker_ethereum_accounting.sql` from the main branch.

thank you!

plz tag me and @web3lyt in the new PR, if you end up migrating 🙏

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**
